src/types/queries.rs

```diff
@@ -324,15 +324,16 @@ pub struct PendingUpdate {
 
 #[derive(Debug, Clone, SerdeSerialize, SerdeDeserialize)]
 #[serde(tag = "updateType", content = "update")]
+#[allow(clippy::large_enum_variant)]
```

Comment:
Did you try addressing the issue instead of disabling this?
If so, I think we should include a comment on why it was decided to ignore the warning, but I think this clippy warning could be worth addressing.

Reply:
Just to add - I think this problem already existed but wasn't detected by clippy because of type families or something. I initially suppressed it to avoid having a breaking change, but I think the impact of the change is very small.